LICENSE

```diff
@@ -20,3 +21,6 @@ http://www.apache.org/licenses/
 A copy is also included in the downloadable source code package
 containing JNA, in file "AL2.0", under the same directory
 as this file.
+
+Commercial license arrangements are negotiable, please e-mail
+twall[at]users[dot]sf[dot]net.
```

Comment:
Neither me, nor others have released their copyright, so there is some interpretation space what this would mean. If it means you can get a copy of JNA and not be bound by ALv2 or LGPLv2+, then I don't see that, but then what does it mean?

I would not put something like this in the license statement.

Reply:
Makes sense, I don't want to speak for @twall about what he meant here, but I changed it to "commercial support".